src/controllers/v1.controller.ts

```
@@ -157,7 +159,15 @@ export class V1Controller {
           blockchainEnforceResult: _enforceResult,
           // eslint-disable-next-line
           blockchainSyncCheck: _syncCheck,
-        } = await this.pocketRelayer.loadBlockchain()
+        } = await loadBlockchain(this.host, this.redis, this.blockchainsRepository, this.defaultLogLimitBlocks).catch(
```

Let's avoid using this catch—we already have a `try` clause here. I think it's quite big so, if we can break it up into steps, that'd be great.

---

Also, this isn't letting me comment on unaffected code, but, this whole logic is inside an `if`, line 155. I think it'd be easier if we checked for the reverse case: if there's *no* LB ID, we can just return (and we should actually throw an error here probably).

---

I agree :)